recipe/meta.yaml

```diff
@@ -16,14 +16,15 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8
```

Comment:
Following the recommendations of the bot https://github.com/conda-forge/statannotations-feedstock/pull/2#issuecomment-2486527586 and referenced documentation https://github.com/conda-forge/cfep/blob/main/cfep-25.md, shouldn't we add this {{ python_min }} above and use it?